sourced/ml/cmd/train_id_split.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import logging
+import os
+import pickle
+
+from sourced.ml.algorithms.id_splitter import set_random_seed, config_keras, \
+    prepare_features, generator_parameters, str2ints, build_train_generator, \
+    prepare_devices, build_rnn, build_cnn, prepare_callbacks, make_lr_scheduler, \
+    report
+
+
+def train_id_split(args: argparse.ArgumentParser):
+    """
+    Pipeline to train a neural network to split identifiers.
+
+    :param args: arguments.
+    :param model: type of neural network used to learn the splitting task.
+    """
+    log = logging.getLogger("train_id_split")
+    config_keras()
+    set_random_seed(args.seed)
+
+    # prepare features
+    X_train, X_test, y_train, y_test = prepare_features(
+                                                    csv_path=args.input,
```

Comment:
The indent looks broken here

Reply:
Because this line could not pass pep8, too long
`X_train, X_test, y_train, y_test = prepare_features(split_identifier_col=args.csv_identifier_split,`

Reply:
give it +4 indent then, do not align at `(`